app/routes/__index/domains/index.tsx

```diff
@@ -1,13 +1,18 @@
 import { AddIcon } from '@chakra-ui/icons';
-import { Button, Container, Flex, Heading, Text } from '@chakra-ui/react';
+import { Button, Container, Flex, Heading, Icon, IconButton, Text } from '@chakra-ui/react';
 import type { LoaderArgs, ActionArgs } from '@remix-run/node';
 import { Link, useNavigate, useSubmit, useTransition } from '@remix-run/react';
 import { typedjson, useTypedLoaderData } from 'remix-typedjson';
+import { FaRedoAlt } from 'react-icons/fa';
 import type { DomainsTableAction } from '~/components/domains-table';
 import DomainsTable from '~/components/domains-table';
-import { getRecordsByUsername } from '~/models/record.server';
+import { getRecordById, getRecordsByUsername } from '~/models/record.server';
 import { requireUsername } from '~/session.server';
 import type { Record } from '@prisma/client';
+import { addDnsRequest, deleteDnsRequest, updateDnsRequest } from '~/queues/dns/dns-flow.server';
```

Comment:
```suggestion
import { deleteDnsRequest, updateDnsRequest } from '~/queues/dns/dns-flow.server';
```

You forgot to remove `addDnsRequest` from the imports